docker/Dockerfile.21.07

```diff
@@ -32,11 +41,25 @@ RUN apt-get update && apt-get install -y locales ninja-build && rm -rf /var/lib/
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
-RUN python3 setup.py install --use-cxx11-abi
+RUN MAX_JOBS=1 python3 setup.py install --use-cxx11-abi
 
 RUN conda init bash
 
-ENV LD_LIBRARY_PATH /opt/conda/lib/python3.8/site-packages/torch/lib:$LD_LIBRARY_PATH
+WORKDIR /opt/trtorch/src
+<<<<<<< HEAD
+RUN cd ./bazel-bin && tar xvf libtrtorch.tar.gz -C /usr/local --strip-components 2 --exclude=LICENSE
+
+ENV LD_LIBRARY_PATH /opt/conda/lib/python3.8/site-packages/torch/lib:${LD_LIBRARY_PATH}
+=======
+RUN mkdir -p /opt/trtorch/lib /opt/trtorch/bin
+RUN cp ./bazel-bin/cpp/bin/trtorchc/trtorchc /opt/trtorch/bin
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorch.so /opt/trtorch/lib
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorchrt.so /opt/trtorch/lib
+RUN cp ./bazel-bin/cpp/api/lib/libtrtorch_plugins.so /opt/trtorch/lib
+
+ENV LD_LIBRARY_PATH /opt/conda/lib/python3.8/site-packages/torch/lib:/opt/trtorch/lib:${LD_LIBRARY_PATH}
+ENV PATH /opt/trtorch/bin:${PATH}
+>>>>>>> 76d9d0448adbf4b224b5219a4f5b38602c188cc5
```

Comment:
merge conflict changes need to be resolved ?

Reply:
Oups, did not commit the resolve. Should be good now.